.rubocop.yml

```diff
@@ -0,0 +1,81 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - vendor/**/*
+    - gemfiles/**/*
+
+
+################
+# Code Metrics #
+################
+Metrics/BlockLength:
+  Exclude:
+    - 'test/schema.rb'
+    - '*.gemspec'
+
+Metrics/AbcSize:
+  Max: 25
+
+Metrics/LineLength:
+  Max: 140
+
+Metrics/MethodLength:
+  Max: 10
+  CountComments: false
+
+
+###############
+# Style Rules #
+###############
+Style/Alias:
+  EnforcedStyle: prefer_alias_method
+
+# ¯\_(ツ)_/¯
+# ʕノ•ᴥ•ʔノ ︵ ┻━┻
+# ( ͡° ͜ʖ ͡°)
+Style/AsciiComments:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Style/DoubleNegation:
+  Enabled: false
+
+Style/EmptyLinesAroundClassBody:
+  EnforcedStyle: empty_lines_except_namespace
+
+Style/EmptyLinesAroundModuleBody:
+  EnforcedStyle: empty_lines_except_namespace
+
+Style/FrozenStringLiteralComment:
+  EnforcedStyle: never
+
+Style/HashSyntax:
+  EnforcedStyle: ruby19
+
+# TODO: replace the [] below with default once a new version of
+#       rubocop comes out, since there is a PR for it:
+#       https://github.com/bbatsov/rubocop/pull/4038
+Style/PercentLiteralDelimiters:
```

Comment:
👍

Note that I just copied these over from acts_as_archival. Feel free to change the hash syntax too. My opinion was we should have a style instead of what that style should be.